README.md

```diff
@@ -795,15 +799,59 @@ opam install sundialsml
 To compile and run the test suite with sundials support:
 
 ```
-make externals-test
+MI_ENABLE_SUNDIALS=1 make test
 ```
 
 To install for the current user:
 
 ```
-make externals-install
+MI_ENABLE_SUNDIALS=1 make install
+```
+
+### Python
+Another optional feature is Python intrinsics, which allow calling Python code
+from MCore. To build the project with Python integration you need to have
+[Python](https://www.python.org) installed on your system. You will also need to
+install any Python libraries you want to use (for example using pip).
+
+In addition, you need the [pyml](https://github.com/thierry-martinez/pyml) OCaml
+Python bindings, available via `opam`:
+
+```
+opam install pyml
+```
+
+To compile and run the test suite with Python support:
+
+```
+MI_ENABLE_PYTHON=1 make test
+```
+
+To install for the current user:
+
+```
+MI_ENABLE_PYTHON=1 make install
+```
+
+The following example shows how to use the intrinsics to to plot an MCore
+sequence using `matplotlib` (assuming it's installed) after sorting it.
+
+```
+let builtins = pyimport "builtins"
+let plt = pyimport "matplotlib.pyplot"
+
+let x = [5.,4.5,4.,1.,1.5]
+let y = pycall builtins "sorted" (x,)
```

Comment:
There might be a point to mention that you can't get values back into mcore. Not sure though, since I got the impression something like it would be added pretty soon.

Reply:
Done!